qiskit/extensions/standard/ms.py

```diff
@@ -28,14 +28,16 @@
 from qiskit.circuit import Gate
 from qiskit.circuit import QuantumCircuit
 from qiskit.circuit import QuantumRegister
+from qiskit.util import deprecate_arguments
 
 
 class MSGate(Gate):
     """Global Molmer-Sorensen gate."""
 
-    def __init__(self, n_qubits, theta):
+    @deprecate_arguments({'n_qubits': 'num_qubits'})
+    def __init__(self, num_qubits, theta, n_qubits=None):  # pylint:disable=unused-argument
```

Comment:
do we need the `*` here to enforce n_qubits can only be passed in by kwarg?

Reply:
I didn't put it there because then the `MSGate` required special treatment in a test where the number of arguments was inferred via the signature of `__init__`. I added the special treatment and `n_qubits` is now keyword only.